js/google_ima.js

```diff
@@ -524,13 +526,33 @@ require("../html5-common/js/utils/utils.js");
       });
 
       /**
-       * Called by the ad manager controller.  Ad Manager Controller lets the module know that an ad should play now.
-       * @public
+       * debouncing the case when user forwarded ahead and 2 or more ads requests simultaneosly issued
```

Comment:
Debouncing the case when a user forwarded ahead and 2 or more ad requests are simultaneously issued